### ISSUE TITLE
Hydra Agent GUI Console

### ISSUE THREAD

**Issue by Guttz** [HTTP-APIs/hydra-python-agent#124]
### I'm submitting a 
- [ ] bug report.
- [x] feature request.

### Description

The agent should have a GUI that enables the user to try and interact with a hydrus server through an interactive interface. Full demo reference mockup:  

![Group h1](https://user-images.githubusercontent.com/11739405/61073311-88d7d900-a3eb-11e9-9607-b97a4b3c0a65.png)

For now, as suggested and settled by @chrizandr in our weekly call, I will focus on the right side of the demo. Thus, the goal now is to give the Agent a 100% functional graphical console to interact with a deployed hydrus server. That means, being able to connect to a hydrus server and having a dynamic UI built based on the ApiDoc of the server that builds valid requests and let the user see the output. 

More info on https://github.com/HTTP-APIs/http-apis.github.io/issues/58

### Do you want to work on this issue?

Yes

**Comment by Guttz** [comment]
One observation @xadahiya asked me to mention was about the framework decision. I've considered mainly two things:

1. The left side of the demo which has the graph which needs a stable and complete library for it not be a problem in the future while generating the Dynamic Graphs. I've checked and tested the best libraries(d3, visjs, sigma.js) and had a much easier time trying to implement what I wanted with React than with Flask. 

2. I have much more previous experiences in Javascript working with frontend and personally prefer it because of all the support around it. So I think this will be more effective considering our time frame and that I'll be responsible for the frontend.

In the end, I've liked vis.js the best for our goal, [it has an extensive library for networks
](https://visjs.github.io/vis-network/docs/network/)  and I've made it work with React and tested some basic graphs like the one below:

![image](https://user-images.githubusercontent.com/11739405/62428836-b78c5b00-b6dc-11e9-9dd1-e244943cd8c0.png)
